lib/node_modules/@stdlib/namespace/alias2pkg/data/data.csv

```diff
@@ -861,11 +997,20 @@ base.sincos,"@stdlib/math/base/special/sincos"
 base.sincospi,"@stdlib/math/base/special/sincospi"
 base.sinh,"@stdlib/math/base/special/sinh"
 base.sinpi,"@stdlib/math/base/special/sinpi"
+base.slice2seq,"@stdlib/slice/base/slice2seq"
```

Comment:
@Planeshifter I am not convinced that base slice utilities should have been exported to the REPL namespace. What was the reason you went through and added all these?